src/lib.rs

```diff
@@ -343,10 +342,9 @@ impl<B: BitBlock> BitVec<B> {
     /// # Examples
     ///
     /// ```
-    /// # #![feature(collections)]
-    /// use std::collections::BitVec;
+    /// use bit_vec::BitVec;
     ///
-    /// let bv = BitVec::from_bytes(&[0b10100000, 0b00010010]);
+    /// let bv = BitVec::<u32>::from_bytes(&[0b10100000, 0b00010010]);
```

Comment:
All examples had to be updated like this. We should probably default new and from_bytes to just a specific block size. Can be generalized later.